bin/p2-fetch-manifest/main.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/square/p2/pkg/logging"
+	"github.com/square/p2/pkg/manifest"
+	"github.com/square/p2/pkg/store/consul"
+	"github.com/square/p2/pkg/store/consul/consulutil"
+	"github.com/square/p2/pkg/store/consul/flags"
+	"github.com/square/p2/pkg/version"
+	kingpin "gopkg.in/alecthomas/kingpin.v2"
+)
+
+const (
+	cmdFetchText = "fetch"
+)
+
+var (
+	cmdFetch     = kingpin.Command(cmdFetchText, "Fetch the manifest for a given pod from the datastore and write it to a file on the disk. Combines well with p2-launch.")
+	fetchTree    = cmdFetch.Flag("tree", "Intent or Reality").Default("intent").Short('t').Enum("intent", "reality")
+	fetchPod     = cmdFetch.Flag("podID", "The pod ID for which to fetch its manifest").Short('i').Required().String()
```

Comment:
I compared this to other in this directory buy using `git grep "Flag.*pod"`. There are many instances of flag `pod`, and I might like to extoll the virtues of consistency.

On the subject of consistency, if this is to remain, perhaps for consistency with `force-restart` it should be `pod-id`...